packages/ui-core/theme/src/lib/components/theme-sidebar/theme-settings/theme-settings.component.html

```
@@ -18,6 +18,44 @@
 		<nb-list-item>
 			<gauzy-layout-selector class="theme"></gauzy-layout-selector>
 		</nb-list-item>
+		<!-- Support / help entries, moved here from the speech-bubble header menu. -->
+		<nb-list-item>
+			<div class="support-links theme">
+				@if (isSupportChatAvailable) {
+					<button nbButton ghost size="small" class="support-link" (click)="openSupportChat()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)